index.bs

```diff
@@ -1085,6 +1087,14 @@ Each {{XRFrame}} has an <dfn for="XRFrame">active</dfn> boolean which is initial
 
 The <dfn attribute for="XRFrame">session</dfn> attribute returns the {{XRSession}} that produced the {{XRFrame}}.
 
+The <dfn attribute for="XRFrame">predictedDisplayTime</dfn> attribute returns the {{DOMHighResTimeStamp}} that represents the point in time at which a rendered frame submitted for this {{XRFrame}} is expected to show up on the devices display.
+
+<div class=note>
+The [=XRFrame/predictedDisplayTime=] is intended to allow rendering an animated XR scene in the state that it should be in when the frame is displayed rather than when the {{XRSession/requestAnimationFrame()}} callback was scheduled or when it was executed.
+
+The [=XRFrame/predictedDisplayTime=] is not intended be used to infer how much time the application has for rendering, as the [=XRFrame/predictedDisplayTime=] is later than when the frame needs to have been submitted.
```

Comment:
I think the concepts of "missing frames" or "making framerate" aren't otherwise described in the spec so far, I'll try to avoid them as so:

> it will miss every single frame and will not be able to make full framerate.

"the [=XR Compositor=] will not be able to make use of the submitted frames, and the application would not hit (make?) target framerate."

Alternatively, I could add definitions for "target framerate", which seems to also be missing (at least in the main spec), "dropping a frame" and "missing a frame"?

Reply:
Since this is a note and not normative, it's ok to use new concepts.